app/models/location.rb

```diff
@@ -374,6 +374,8 @@ def available_from
   end
 
   def status
+    return :incomplete if [postcode, units, type_of_unit, mobility_type].any?(&:blank?)
```

Comment:
Could do something similar to this: https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/pull/1003#discussion_r1025176900

Reply:
Thanks @kosiakkatrina. From what I can see, schemes have a method to validate whether they're confirmed (see app/models/scheme.rb#199). However there doesn't seem to be an equivalent validation function for locations - it just seems like if a scheme is confirmed, then its locations are marked as confirmed too (see app/controllers/schemes_controller.rb#107). Assuming that's correct, do you think `confirmed` is actually that useful for locations currently? I think perhaps not, and that we should raise a new ticket to update how locations get marked as confirmed. Let me know your thoughts! Thanks 🙂

Reply:
@kosiakkatrina any thoughts on this?

Reply:
Hiya, sorry for getting to this late. 
So the scheme gets marked as confirmed from the check your answers page for that scheme and at that point all of the locations associated with it would be completed (because we'd throw validation errors when adding a location otherwise).
If a new location gets added to that scheme afterwards it wouldn't be marked as confirmed until the scheme is saved from check your answers page again. Which again, would not work unless the location is completed.
So I think using confirmed should work fine

Reply:
@kosiakkatrina thanks, I understand that.

However I've implemented this change and my tests are now failing - instead of returning `incomplete` the status method is now returning `active`, so it doesn't seem to be working as expected. Can't work out why this might be. Any thoughts?